RELEASING.md

```diff
@@ -0,0 +1,64 @@
+# Releasing
+
+Checklist before starting:
+* Make sure that your local master branch is in sync with upstream.
+* Make sure that package-lock.json is up-to-date, and generated with the latest npm
+  LTS version.
```

Comment:
NPM doesn't do LTS releases. I'm not sure what our policy for package-lock.json should be, but since it doesn't affect releases (only the development environment), it shouldn't be a part of this document.

Reply:
Agree.